albumentations/augmentations/geometric/transforms.py

```diff
@@ -143,11 +152,23 @@ def __init__(
         ],
         keypoint_remapping_method: Literal["direct", "mask"],
         p: float,
+        border_mode: Literal[
+            cv2.BORDER_CONSTANT,
+            cv2.BORDER_REPLICATE,
+            cv2.BORDER_REFLECT,
+            cv2.BORDER_WRAP,
+            cv2.BORDER_REFLECT_101,
+        ] = cv2.BORDER_CONSTANT,
+        fill: tuple[float, ...] | float = 0,
+        fill_mask: tuple[float, ...] | float = 0,
     ):
         super().__init__(p=p)
         self.interpolation = interpolation
         self.mask_interpolation = mask_interpolation
         self.keypoint_remapping_method = keypoint_remapping_method
+        self.border_mode = border_mode
+        self.fill = fill
+        self.fill_mask = fill_mask
```

Comment:
border mode, fill and fill mask do not have material effects in this transform, let's not add them.

Reply:
My use-case involves extreme deformations (alpha=7000, sigma=60).
In this case, significant portions around the edges of the image sample outside of the original image after stretching.
I have used the border_mode argument for choosing whether these parts are filled with a solid color or reflected, (which is a very meaningful visual difference), and I have found this to be an effective augmentation.
Without these arguments, I would not be able to use this transform.

Reply:
I see. Did not think about extreme deformations. Sounds good.